processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -373,6 +429,12 @@ private void makeMetricsColumns(
                   .serializerBuilder()
                   .withByteOrder(IndexIO.BYTE_ORDER)
                   .withDelegate((LongColumnSerializer) writer)
+                  .withNullValueBitmapWriter(createNullRowsBitmapWriter(
```

Comment:
@nishantmonu51 wondering if it makes more sense to blend the logic of bitmaps writer with the metric writer itself hence you have to manage only one writer instead of having `LongColumnSerializer writer`  and the `NullRowsBitmapWriter`, this can be done by creating the bitmap writer within the metric writer itself. ?